packages/profiles/.prettierignore

```diff
@@ -1,16 +1 @@
-# Copyright (c) 2020 Broadcom.
```

Comment:
Yah. Let's do that. 👍

Reply:
@jellypuno Nikolay checked in with me, he said:

_I just wanted to say, that it is not required to have copyright headers above the development configurations files, like .gitignore, because we have to protect the code copyrights (.js/.ts) files and some custom configurations. Also, these files will not be part of the distribution vsix file and the repo itself contains license and notice files.
I recommend you to discuss it with Vit (I think he will be back today) and if he thinks the same - you can proceed with the release, if it is a showstopper._

So, I will discuss with Vit, if he is back today, and we can move forward possibly.

Reply:
This is an ignore file for prettier, there is no point in having a copyright statement there.

Reply:
@VitGottwald @jellypuno The only ones I see there that aren't something like .gitignore or package.json are [these test schema files](https://github.com/eclipse/che-che4z-explorer-for-endevor/tree/77382cce63c58352fa14da237bef142f8167b6c5/packages/explorer-for-endevor/src/services/__tests__/__fixtures__)

Should they be copywritten?